### ISSUE TITLE
Illegal instruction (core dumped)

### ISSUE THREAD

**Issue by RafieeAshkan** [takuseno/d3rlpy#23]
Hi
I just ran the Atari 2600 test case in Readme but when running I get the following error

`Epoch 0:   0%|                                        | 0/24760 [00:00<?, ?it/s]Illegal instruction (core dumped)`

Any idea why I am getting the error? the only change I made was dropping `use_gpu=True`

**Comment by takuseno** [comment]
@RafieeAshkan Hi! Thank you for reporting the issue! This seems that the binary module raises the error. Could you share your environment (CPU, OS, Python version etc).

Thank you!

**Comment by RafieeAshkan** [comment]
@takuseno Many thanks for looking into resolving the issue. Here are the info you asked:

OS: Ubuntu 18.04
CPU: Intel® Core™ i7-8650U CPU @ 1.90GHz × 8
Memory: 16 GiB
Python: 3.7.5

please let me know if you need any further info

**Comment by takuseno** [comment]
@RafieeAshkan Thanks for the response! It looks pretty normal... Which did you install this via `pip` or `conda`? I appreciate your cooperation!

**Comment by RafieeAshkan** [comment]
@takuseno I installed it with `pip`

**Comment by RafieeAshkan** [comment]
@takuseno was wondering if there has been any update on the issue. I am really keen to use your great package for offline RL. Thanks

**Comment by takuseno** [comment]
@RafieeAshkan Sorry for the inconvenient... I cannot reproduce this issue. If it's possible fo you, I'd like you to try building from source.

```
$ git clone git@github.com/takuseno/d3rlpy
$ cd d3rlpy
$ pip install Cython
$ pip install -e .
```

**Comment by RafieeAshkan** [comment]
@takuseno I just installed from source and all is working perfectly fine. I don't know if you want to close the issue or still check the `pip` source issue. 
Many thanks for the great package and your support.

**Comment by takuseno** [comment]
I guess the fix made for #59 solves this too. Please reopen this when you meet the same error after v0.71 release.
